python/dgl/transform.py

```diff
@@ -18,7 +18,9 @@
 from . import utils
 from .base import EID, NID
 from . import ndarray as nd
-
+from .partition import metis_partition_assignment as hetero_metis_partition_assignment
+from .partition import partition_graph_with_halo as hetero_partition_graph_with_halo
+from .partition import metis_partition as hetero_metis_partition
```

Comment:
i think we can just remove the old implementations.

Reply:
Preserved for now to debug further end2end examples